packages/react-heartwood-components/src/components/List/components/ListItem/ListItem.js

```diff
@@ -109,22 +121,50 @@ const ListItem = (props: Props) => {
 						/>
 					)}
 					{avatar && <Avatar image={avatar} alt={title} />}
+					{checkboxId && (
+						<Fragment>
+							{selectableType === 'checkbox' && (
+								<Checkbox id={checkboxId} {...checkboxProps} />
+							)}
+							{selectableType === 'radio' && (
+								<Radio id={checkboxId} {...checkboxProps} />
+							)}
+						</Fragment>
+					)}
```

Comment:
@jtini `checkboxId` and `selectableType` are both optional but don't seem mutually exclusive. e.g. if I pass `selectableType` but not a `checkboxId` this wouldn't render my checkbox. Similarly I'm wondering if the `checkboxId` should be `selectableId` to eliminate confusion as to whether it's applies to both checkbox/radio

Reply:
The non-mutual-exclusivity I think is fine, because it prevents id-less checkboxes/radios from being rendered that would cause problems. I agree about naming though, I'll change `checkboxId` and `checkboxProps` to `selectableId` and `selectableProps` respectively.

Reply:
👍